go.mod

```diff
@@ -8,19 +8,56 @@ require (
 	github.com/go-openapi/runtime v0.19.23
 	github.com/jmoiron/jsonq v0.0.0-20150511023944-e874b168d07e
-	github.com/juju/errgo v0.0.0-20140925100237-08cceb5d0b53 // indirect
-	github.com/mattn/go-colorable v0.0.9 // indirect
 	github.com/mattn/go-isatty v0.0.3
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/mapstructure v1.3.2
-	github.com/oracle/oci-go-sdk/v48 v48.0.0
+	github.com/oracle/oci-go-sdk/v65 v65.28.1
 	github.com/spf13/viper v1.6.2
 	github.com/urfave/cli v1.20.0
-	github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f // indirect
-	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	github.com/xeipuuv/gojsonschema v0.0.0-20180618132009-1d523034197f
-	golang.org/x/sys v0.0.0-20220804214406-8e32c043e418
+	golang.org/x/sys v0.6.0
 	gopkg.in/yaml.v2 v2.3.0
 )
 
-go 1.14
+require (
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
+	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
+	github.com/asaskevich/govalidator v0.0.0-20200428143746-21a406dcc535 // indirect
+	github.com/fsnotify/fsnotify v1.4.7 // indirect
+	github.com/go-openapi/analysis v0.19.10 // indirect
+	github.com/go-openapi/errors v0.19.8 // indirect
+	github.com/go-openapi/jsonpointer v0.19.3 // indirect
+	github.com/go-openapi/jsonreference v0.19.3 // indirect
+	github.com/go-openapi/loads v0.19.5 // indirect
+	github.com/go-openapi/spec v0.19.8 // indirect
+	github.com/go-openapi/strfmt v0.19.8 // indirect
+	github.com/go-openapi/swag v0.19.11 // indirect
+	github.com/go-openapi/validate v0.19.12 // indirect
+	github.com/go-stack/stack v1.8.0 // indirect
+	github.com/gofrs/flock v0.8.1 // indirect
+	github.com/golang/mock v1.4.4 // indirect
+	github.com/hashicorp/hcl v1.0.0 // indirect
+	github.com/juju/errgo v0.0.0-20140925100237-08cceb5d0b53 // indirect
+	github.com/magiconair/properties v1.8.1 // indirect
+	github.com/mailru/easyjson v0.7.1 // indirect
+	github.com/mattn/go-colorable v0.0.9 // indirect
+	github.com/pelletier/go-toml v1.7.0 // indirect
+	github.com/sony/gobreaker v0.5.0 // indirect
+	github.com/spf13/afero v1.1.2 // indirect
+	github.com/spf13/cast v1.3.0 // indirect
+	github.com/spf13/jwalterweatherman v1.0.0 // indirect
+	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/subosito/gotenv v1.2.0 // indirect
+	github.com/xeipuuv/gojsonpointer v0.0.0-20180127040702-4e3ac2762d5f // indirect
+	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
+	go.mongodb.org/mongo-driver v1.4.2 // indirect
+	golang.org/x/crypto v0.0.0-20201016220609-9e8e0b390897 // indirect
+	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b // indirect
+	golang.org/x/text v0.3.3 // indirect
+	gopkg.in/ini.v1 v1.51.0 // indirect
+)
+
+replace github.com/fnproject/fn_go v0.8.7 => /Users/sunny/Functions/fn_go
+replace github.com/oracle/oci-go-sdk/v65 v65.28.1 => /Users/sunny/Functions/oci-go-sdk-preview/oci-go-sdk-with-edit/oci-go-sdk
```

Comment:
These replace directives need to be removed

Reply:
Same for the fn_go, once it is released then we can update it here.